### ISSUE TITLE
Crash when trying to create a new world

### ISSUE THREAD

**Issue by Sunconure11** [KubeJS-Mods/KubeJS#501]
### Minecraft Version

1.19.2

### KubeJS Version

1902.6.0-build.79

### Rhino Version

1902.2.1-build.242

### Architectury Version

6.2.46

### Forge/Fabric Version

Quilt 0.17.5-beta3

### Describe your issue

Whenever I try to make a new world, I get a crash blaming your mod

### Crash report/logs

https://pastebin.com/8w4ALpkJ

**Comment by Abalieno** [comment]
Happens on 1.18.2 fabric after I updated to the latest kubejs/rhino released earlier today.

**Comment by ColdTearsYY** [comment]
Same error on my 1.19.2 fabric server with same message.

**Comment by otavio-silva** [comment]
I'm having the same issue and my post was closed and marked as "not planned", does that mean that this error is not going to be fixed or it was because the post was a duplicate?

**Comment by Sunconure11** [comment]
Looks like it is because it is a duplicate of this issue. I had also brought it up on the Discord run by this dev.

**Comment by staviq** [comment]
Same problem here, both builds "KubeJS Fabric 1902.6.0-build.86" and "KubeJS Fabric 1902.6.0-build.79"

result in crash with:

`Failed to load datapacks, can't proceed with server load. You can either fix your datapacks or reset to vanilla with --safeMode
java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot read field "logErroringRecipes" because "dev.latvian.mods.kubejs.server.ServerSettings.instance" is null`


When trying to add this mod to an existing server / word.

Interestingly enough, this does not happen when starting the client, untill i click the singleplayer button

**Comment by ChiefArug** [comment]
> I'm having the same issue and my post was closed and marked as "not planned", does that mean that this error is not going to be fixed or it was because the post was a duplicate?

Yes, when you go to close an issue github has a few words describing the two close states, and duplicate is one of the ones for not planned.

**Comment by otavio-silva** [comment]
> > I'm having the same issue and my post was closed and marked as "not planned", does that mean that this error is not going to be fixed or it was because the post was a duplicate?
> 
> Yes, when you go to close an issue github has a few words describing the two close states, and duplicate is one of the ones for not planned.

Well, I know that _now_. When people say "not planned", they usually mean "not going to be implemented", and not "this is a duplicate". A "closed as duplicate" would be way more clarifying.

**Comment by MSandro** [comment]
I have the same issue in my 1.19.2 modpack.

**Comment by TheButterbrotMan** [comment]
https://github.com/KubeJS-Mods/KubeJS/issues/507
